client/src/components/styles/CommonStyle.scss

```diff
@@ -1,5 +1,11 @@
 @import "variables";
 
+.wrapper {
```

Comment:
pls rename it -- general.scss or style.scss will be better

Reply:
Done